strongbox-security/strongbox-authentication-registry/src/test/java/org/carlspring/strongbox/authentication/ConfigurableProviderManagerTest.java

```diff
@@ -68,6 +75,43 @@
     @PersistenceContext
     protected EntityManager entityManager;
 
+    @Test
+    @Transactional
+    public void testShouldNotAllowExternalUsersDublicate() 
```

Comment:
Cosmetic: There's a small typo here:
```suggestion
    public void testShouldNotAllowExternalUserDuplicate() 
```